test/unit/capsize_deployer_test.rb

```diff
@@ -193,17 +193,17 @@ def test_handling_of_prompt_configuration
     assert stage_with_prompt.deployment_possible?, stage_with_prompt.deployment_problems.inspect
 
     # add a config value that wants a promp
-    stage_with_prompt.configuration_parameters.build(:name => 'password', :prompt_on_deploy => 1).save!
+    stage_with_prompt.configuration_parameters.create(:name => 'password', :prompt_on_deploy => 1)
```

Comment:
Shouldn't this be `create!` to stick with the existing `save!` semantics? You're also not checking the return value, which means you are silently discarding an error condition.